src/app/modules/spcp/spcp.util.ts

```diff
@@ -180,14 +180,14 @@ export const createCorppassParsedResponses = (
     {
       _id: '',
       question: 'CorpPass Validated UEN',
-      fieldType: 'authenticationCp' as BasicField,
+      fieldType: BasicField.ShortText,
       isVisible: true,
       answer: uinFin,
     },
     {
       _id: '',
       question: 'CorpPass Validated UID',
-      fieldType: 'authenticationCp' as BasicField,
+      fieldType: BasicField.Nric,
```

Comment:
Should this just be ShortText also? It's not really an NRIC

Reply:
Isn't it an NRIC? I also did it for consistency with the decryption worker - see process-decrypted-content.js, L31